packages/config/config/devices/0x010f/templates/fgs-2xx.json

```diff
@@ -0,0 +1,302 @@
+// Template for FGS-211, FGS-212, FGS-221 and FGS-222
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 16
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 16
+		},
+		"3": {
+			"label": "Controller Updates",
+			"description": "Reports state of devices.",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "fibarogroup_templates.json#enable_disable_all_on_off"
+		},
+		"3": {
+			"label": "Auto off relay after specified time",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Manual override disabled",
+					"value": 0
+				},
+				{
+					"label": "Manual override enabled",
+					"value": 1
+				}
+			]
```

Comment:
The label does not seem to match the options

Reply:
Existing parameter https://github.com/zwave-js/node-zwave-js/blob/910155c88ffb2f8ab6ddce6adc2c0dbb57a135f6/packages/config/config/devices/0x010f/fgs221_2.1-2.3.json#L101-L121

Reply:
That does not mean it is good :)